README.md

```diff
@@ -66,15 +66,71 @@ https://github.com/btcsuite/btcwallet/releases
 
 ### Windows/Linux/BSD/POSIX - Build from source
 
-- If necessary, install Go according to the installation instructions
-  here: http://golang.org/doc/install.  It is recommended to add
-  `$GOPATH/bin` to your `PATH` at this point.
+Building or updating from source requires the following build dependencies:
+
+- **Go 1.5 or 1.6**
+
+  Installation instructions can be found here: http://golang.org/doc/install.
+  It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
+
+- **Glide**
+
+  Glide is used to manage project dependencies and provide reproducable builds.
+  To install:
+
+  `go get -u github.com/Masterminds/glide`
+
+Unfortunately, the use of `glide` prevents a handy tool such as `go get` from
+automatically downloading, building, and installing the source in a single
+command.  Instead, the latest project and dependency sources must be first
+obtained manually with `git` and `glide`, and then `go` is used to build and
+install the project.
+
+**Getting the source**:
+
+For a first time installation, the project and dependency sources can be
+obtained manually with `git` and `glide` (create directories as needed):
+
+```
+git clone https://github.com/btcsuite/btcwallet $GOPATH/src/github.com/btcsuite/btcwallet
+cd $GOPATH/src/github.com/btcsuite/btcwallet
+glide install
```

Comment:
Yes, that is exactly what I was going to suggest along with a pseudo heading note.  Perhaps:

------

To build and install `btcwallet` and all helper commands (in the `cmd`
directory) to `$GOPATH/bin/`, as well as installing all compiled non-vendored
packages to `$GOPATH/pkg/` (**use this if you are unsure which command to
run**):

```
go install $(glide novendor)
```

**NOTE**: The following are alternatives and do not need to be
executed unless you specifically want the different behavior
described.

* To build a `btcwallet` executable and place it in the current directory:

```
go build
```
**OR**

* To build a `btcwallet` executable and install it to `$GOPATH/bin/btcwallet`:

```
go install
```

-----

Another issue I see is that the `$()` syntax doesn't work under Windows `cmd`.  This should call that out, in my opinion.